eslint.config.js

```diff
@@ -262,6 +262,15 @@ export default [
 		}
 	},
 
+	{
+		files: [
+			"build/**/*.mjs"
+		],
+		languageOptions: {
+			sourceType: "module"
+		}
```

Comment:
Is this necessary? Doesn't ESLint infer it from the extension?

Reply:
Yes and no. The above `build/**/*` overwrites the default to be commonjs, which all `.js` files are in the build folder unless they have the `.mjs` extension. The other way I was thinking to write this was to only apply the above fules only to `.js` and `.cjs` files, but then I'd have to copy all the other rules to `.mjs` in a separate rule. That said, I think I know a better way, which is to not set sourceType above, but change this rule to set `.js` files to `commonjs`.